### PR TITLE
chmod the OPW envfile for better security

### PR DESCRIPTION
**Pull request by spencergilbert**

### CODE REVIEW

File: install_script_op_worker1.sh [hunk 1]
```diff
@@ -505,6 +505,8 @@ if [ ! -e "$pipeline_file" ]; then
   fi
 fi
 
+$sudo_cmd chmod 640 "$env_file"
```
Comment (chouetz) [inline install_script_op_worker1.sh:508]: Not sure if I'm paranoid or not, should you `chmod` before appending any secret to the file?

Comment (bkabrda) [reply install_script_op_worker1.sh:508]: Agreed. Let's first create the file, then chmod and only then start writing values into it.

Comment (spencergilbert) [reply install_script_op_worker1.sh]: Moved in https://github.com/DataDog/agent-linux-install-script/pull/72/commits/5b520df72356b3dddcd4219a0cef7cbb262c7fec

**Review by bkabrda — APPROVED** [review]

**Comment by spencergilbert** [comment]
Force pushed as I rebased on `main` to pass CI.

**Pull request merged**